app/views/organisations/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<% content_for :title, "Departments, agencies and public bodies - GOV.UK" %>
+
+<h1>Departments, agencies and public bodies - GOV.UK</h1>
+<% @content_item["details"].each do |organisation_id, organisation| %>
```

Comment:
I think we might want to split this out so that we don't automatically iterate through the content item's details.  If someone adds something to the details of the content item, then this page will try and render it, which could be interesting!  

Also, we want to be overt about which order we display things in, which might not be the same order that things are sent from the content store.  Splitting the innards into a partial, but making separate calls for each type that we want to show, in the order that we want to show them should reduce our risk of doing something inadvertent.